packages/universe-api/src/checkApiKey.ts

```diff
@@ -1,28 +1,45 @@
 import { prismaClient } from "@cfce/database"
+import { UserType } from "@cfce/types"
 
 const officialApiKey = process.env.OFFICIAL_CFCE_API_KEY
 
 const checkApiKey = async (
   apiKey: string | null,
-  id?: string,
+  options?: {
+    userId?: string
+    orgId?: string
+    adminOnly?: boolean
+    devOnly?: boolean
+  },
 ): Promise<boolean> => {
-  if (!apiKey) {
-    return false
-  }
+
+  const { userId, orgId, adminOnly, devOnly } = options || {}
+
+  //if (!apiKey) {
+  //  return false
+  //}
 
   if (apiKey === officialApiKey) {
     return true
   }
 
+  if (devOnly && process.env.NEXT_PUBLIC_APP_ENV !== "development") {
+    return false
+  }
+
   const user = await prismaClient.user.findUnique({
-    where: { api_key: apiKey },
+    where: { api_key: apiKey || '12345678-1324-1234-1324-123456789ABC' } // Never find it if no key
```

Comment:
Who knows, maybe one person will get lucky 😂

Reply:
If the source code is open, people would just use that api key and get access. Will remove it

Reply:
Oh yeah, good point!